pom.xml

```diff
@@ -59,6 +59,7 @@
 		<h2.version>2.1.212</h2.version>
 		<slf4j-api.version>1.7.36</slf4j-api.version>
 		<jakarta-validation-api.version>2.0.2</jakarta-validation-api.version>
+		<lombok.version>1.18.24</lombok.version>
```

Comment:
🤔 FYI: I will try to prepare an extra PR later on to show how to use spring boot bom without using it as a parent to avoid keeping versions such this one.

Reply:
Sure, good idea.